src/site/content/en/animations/animations-examples/index.md

```diff
@@ -0,0 +1,346 @@
+
+## Wizard loading animation
+
+{% IFrame {
+  src: 'https://codepen.io/iamlark/embed/jYzYJg?height=458&theme-id=light&default-tab=result',
+  title: 'Only CSS Loader - Wizard by Guilmain Dorian'
+}
+%}
+
+This loading animation is built entirely with CSS.
+The image plus all of the animation has been created in CSS and HTML,
+no images or JavaScript.
+To understand how it was created and how well it performs you can use Chrome DevTools.
+
+With the animation running, open the Performance tab in Chrome DevTools and record a few seconds of the animation.
+You should see in the Summary that the browser is not doing any Layout or Paint operations when running this animation.
+
+<figure class="w-figure">
+  <img src="./wizard-summary.jpg" alt="Summary in DevTools">
+  <figcaption class="w-figcaption">
+    The summary after profiling the wizard animation.
+  </figcaption>
+</figure>
+
+To find out how this animation was achieved without causing layout and paint,
+inspect any of the moving elements in Chrome DevTools.
+For example select the triangle,
+and watch how the box of the element transforms during its journey into the air,
+as it spins, and then returns to the start position.
+
+<figure class="w-figure">
+  <video controls autoplay loop muted class="w-screenshot">
+    <source src="https://storage.googleapis.com/web-dev-assets/triangle.mp4" type="video/webm">
+  </video>
+  <figcaption class="w-figcaption">
+    Video showing how we can track the path of the triangle in Chrome DevTools.
+  </figcaption>
+</figure>
+
+With the element still selected look at the Styles Panel.
+There you can see the CSS which draws the shape of the triangle,
+and the animation being used.
+
+The triangle is created by using the `::after` pseudo-element to add generated content,
+using borders to create the shape.
+
+```css
+.triangle {
+    position: absolute;
+    bottom: -62px;
+    left: -10px;
+    width: 110px;
+    height: 110px;
+    border-radius: 50%;
+}
+
+.triangle::after {
+    content: "";
+    position: absolute;
+    top: 0;
+    right: -10px;
+    width: 0;
+    height: 0;
+    border-style: solid;
+    border-width: 0 28px 48px 28px;
+    border-color: transparent transparent #89beb3 transparent;
+}
+```
+
+{% Glitch 'animation-breakdown1-1' %}
+
+{% Aside %}
+You can find out more about making shapes with borders and generated content in
+[The Shapes of CSS](https://css-tricks.com/the-shapes-of-css/).
+{% endAside %}
+
+The animation is added with the following line of CSS,
+
+```css
+animation: path_triangle 10s ease-in-out infinite;
+```
+
+Find the keyframes by scrolling down the Style Panel.
+There you will find that the animation is created by using `transform` to change the position of the element and rotate it.
+The `transform` property is one of the properties described in the [Animations Guide](/animations-guide),
+which does not cause the browser to do layout or paint operations.
```

Comment:
```suggestion
which does not cause the browser to do layout or paint operations (which are the main causes of slow animations).
```
Perhaps we shouldn't assume they've read our other stuff and therefore we should be careful about avoiding (what they might perceive as) technical jargon. In this case we might quickly clarify why exactly they should be wary of paint or layout